docs/release_notes/release_1_0_3.rst

```diff
@@ -8,6 +8,18 @@ This release is currently in progress.
 New features
 ------------
 
+ - The device setup can be changed at runtime through the control server. It is not possible to switch to another device, only setups of the same device can be used. To query available setups:
```

Comment:
@MichaelWedel is setups ubiquitous enough to mention here without reference to what a setup is? I've seen this in a few places.